apps/nextra/pages/en/network/blockchain/execution.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Parallel Execution"
+---
+
+# Execution
+
+In blockchains, execution refers to the process by which validators 
+run and apply smart contract transactions from ordered blocks to update the blockchain’s state.
+
+It is crucial because validators must agree on the same final state, post-execution,
+ensuring consistency and security across the network.
+
+Efficient execution, particularly parallel execution, is key to scaling blockchain performance and throughput.
+
+## Why Parallel Execution?
+
+![](https://miro.medium.com/v2/resize:fit:4800/format:webp/0*qG0KRe1KzsP8Odth)
+
+Smart contract execution is a major throughput bottleneck for blockchains. 
+After proposing blocks and agreeing on their order, validators have to execute the 
+transactions in the ordered blocks. Crucially, validators must arrive at the same final state, 
+which must correspond to some sequential execution of the transactions. 
+Due to the lack of a foundational solution, current blockchains either execute 
+sequentially or require an embarrassingly parallel workload (i.e. no inner conflicts) 
+for improved performance. Sequential execution does not scale well, and assuming that 
+all transactions are commutative is unrealistic when considering a broad range of smart contracts. 
+
+In fact, transactions in a blockchain can have a significant number of access conflicts due to 
+potential performance attacks, accessing popular contracts 
+(e.g. due to economic opportunities such as auctions and arbitrage).
+
+## Block-STM
+
+We designed and implemented a highly efficient, multi-threaded, in-memory parallel 
+execution engine that can execute over 160k non-trivial Move transactions per 
```

Comment:
Best to double check with marketing if we still want to claim this number

Reply:
Sounds good, mmm let me omit for now and we can add it back in later if its OK